docs/publishing.md

```diff
@@ -25,16 +25,18 @@ The files under scripts use [shelljs/shx](https://github.com/shelljs/shx) and
 [shelljs/shelljs](https://github.com/shelljs/shelljs) to write scripts in a
 portable manner across platforms.
 
+1. `git checkout -t origin release/vxx.yy.zz`
+1. `npm install`
+1. `export SALESFORCEDX_VSCODE_VERSION=xx.yy.zz` (must match the branch version)
+1. `scripts/publish.js`
+
 It is possible to run each step manually as illustrated below.
 
 ## Packaging as .vsix
 
 ### Prerequisite
 
-* Lerna is properly installed (`npm install -g lerna@2.0.0`).
-* shelljs is properly installed (`npm install -g shelljs`).
-* You have a clean workspace. You can use `git clean -xfd` to get to a pristine
-  stage.
```

Comment:
Shouldn't we keep this but modify it to leave top-level node_modules (git clean -xfd -e node_modules) ?

Reply:
The publish scripts actually runs `git clean -xfd -e node_modules` for you. That's why I avoided doing it here.